test/apps/iodemo/io_demo.cc

```diff
@@ -785,13 +786,71 @@ class P2pDemoCommon : public UcxContext {
         return (data_size + chunk_size - 1) / chunk_size;
     }
 
+    static void handle_validate_error(const std::exception &e,
+                                      const UcxConnection *conn) {
+        LOG << "\"" << e.what() << "\" detected on " << conn->get_log_prefix()
+            << " (status=" << ucs_status_string(conn->ucx_status()) << ")";
+        abort();
+    }
+
+    class ValidateError : public std::exception {
+    public:
+        ValidateError() throw() {
```

Comment:
here and below - why throw() is needed? I would suggest to avoid using deprecated syntax if possible

Reply:
compiling it w/o c++11 support requires it